### PR TITLE
Added 'appraisal'-integration changes

### PR DESCRIPTION
**Pull request by jasnow**
Added 'appraisal'-integration changes to 3 files (Rakefile, .travis.yml, README.rdoc).

Checked TravisCI: https://travis-ci.org/jasnow/setler/builds/87804657 and it was green.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+require "rubygems"
+require "bundler/setup"
+
```
Comment (ckdake) [inline Rakefile]: Lets keep '/" style consistent here. Either change 'em all or follow what is in there.

File: Rakefile [hunk 2]
```diff
@@ -8,4 +11,8 @@ Rake::TestTask.new(:test) do |test|
   test.verbose = true
 end
 
-task default: [:test]
+if !ENV["APPRAISAL_INITIALIZED"] && !ENV["TRAVIS"]
+  task default: :appraisal
+else
+  task default: [:test]
+end
```
Comment (ckdake) [inline Rakefile]: newline please

**Comment by jasnow** [comment]
Anything else? Thanks....

**Comment by ckdake** [comment]
Oh if only github sent email notifications for new commits on a PR...

**Comment by jasnow** [comment]
I have the same problem. I try to Watch and Starr the repo to see if that helps.

Thanks for letting me help out on the project. Cheers.....

**Pull request merged**